src/components/common/ListPeopleAndRoles.vue

```diff
@@ -83,6 +88,9 @@
             <p>{{ orgPerson.officer.email }}</p>
           </v-col>
 
+          <!-- conditional empty alignment column -->
+          <v-col v-if="showAlignmentColumn" />
```

Comment:
I chatted with Ethan and this is the compromise we decided on.

Sample incorporation - Add People and Roles (5 columns):
![image](https://user-images.githubusercontent.com/10002889/221967391-3f2859c8-c48c-4168-9d22-3b45519a634e.png)

Sample incorporation - Review and Confirm (4 columns):
![image](https://user-images.githubusercontent.com/10002889/221967582-4d623bbf-739b-4ac7-b083-17c16366e6ce.png)

Sample restoration - Add Applicant Information (4 columns):
![image](https://user-images.githubusercontent.com/10002889/221967721-0722fb56-70a8-41cd-97c6-1d442db2f135.png)

Sample restoration - Review and Confirm (3 columns + 1 empty column):
![image](https://user-images.githubusercontent.com/10002889/221967891-5800d1f1-05c7-4ed2-ba4f-7f44fb19dee4.png)